ogr/ogrsf_frmts/openfilegdb/ogropenfilegdbdatasource.cpp

```diff
@@ -243,21 +245,37 @@ int OGROpenFileGDBDataSource::Open(const GDALOpenInfo *poOpenInfo )
                 continue;
             }
 
-            const OGRField* psField = oTable.GetFieldValue(0);
+            const OGRField* psField = oTable.GetFieldValue(idxName);
             if( psField != nullptr )
             {
                 aosTableNames.push_back(psField->String);
-
-                if( strcmp(psField->String, "GDB_Items") == 0 )
+                if( strcmp(psField->String, "GDB_SpatialRefs") == 0 )
+                {
+                    // Normally a00000003
+                    m_osGDBSpatialRefsFilename = CPLFormFilename(m_osDirName,
+                            CPLSPrintf("a%08x.gdbtable", i + 1), nullptr);
+                }
+                else if( strcmp(psField->String, "GDB_Items") == 0 )
                 {
+                    // Normally a00000004
```

Comment:
For reference, I've seen cases where this **isn't** "a...04", so it's not just theoretically possible.

Reply:
interesting. I'll add a note